pkg/sql/sem/builtins/fingerprint_builtin_test.go

```diff
@@ -89,10 +88,11 @@ func TestFingerprint(t *testing.T) {
 	}
 
 	fingerprint := func(t *testing.T, startKey, endKey string, startTime, endTime hlc.Timestamp, allRevisions bool) int64 {
-		decimal := eval.TimestampToDecimal(endTime)
+		aost := endTime.AsOfSystemTime()
 		var fingerprint int64
-		query := fmt.Sprintf(`SELECT * FROM crdb_internal.fingerprint(ARRAY[$1::BYTES, $2::BYTES], $3, $4) AS OF SYSTEM TIME %s`, decimal.String())
-		require.NoError(t, sqlDB.QueryRow(query, roachpb.Key(startKey), roachpb.Key(endKey), startTime.GoTime(), allRevisions).Scan(&fingerprint))
+		query := fmt.Sprintf(`SELECT * FROM crdb_internal.fingerprint(ARRAY[$1::BYTES, $2::BYTES],$3::DECIMAL, $4) AS OF SYSTEM TIME '%s'`, aost)
```

Comment:
so this test is for the decimal overload, do we have a test for the TimestampTZ one?

Reply:
yeah, there's a test below this one in this package and plenty of other unit tests in other packages use the timestamp overload.